### DISCUSSION TITLE
SAP2000 Adapter Error for SAP2000v25

### DISCUSSION THREAD

**Discussion by jaredmytra** [BHoM/BHoM#1596] [Q&A]
@enarhi 
The following error shows when trying to use SAP2000 Adapter to push geometry to SAP2000v25:
1. This component failed to run properly.
- Error: Method not found: 'System.Object System.Runtime.InteropServices.Marshal.GetActiveObject(System.String)'.
- Occured in  BH.Adapter.SAP2000.SAP2000Adapter..ctor(String filePath, Boolean active)
     called from BH.UI.Base.Caller.Run(List`1 inputs)
- Are you sure you have the correct type of inputs? Check their description for more details.

This is true whether I have SAP2000 open or not

**Comment by samuelbasimalla** [comment]
@jaredmytra 
I have recently used SAP2000 Adapter for SAP2000v24, which did not give me any errors. 
Can you tell which BHoM version you are using? It would also be helpful if you can share test script which is giving you this error, to help see if this error is specific to v25.

**Reply by rezzzq** [reply]
I am getting the same error with ETABS adapter for all etabs version: 16,17,18. I am using BHoM version 7.3 
![image](https://github.com/user-attachments/assets/7e48b88d-c61c-457d-90cf-1a76e7179512)

**Reply by amrzk** [reply]
Similar error here as well BHoM version 8.1
![image](https://github.com/user-attachments/assets/9521f92e-136e-4dfa-b1c0-56595a685f2a)
